components/scream/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp

```diff
@@ -0,0 +1,177 @@
+#include <iostream>
+#include "physics/rrtmgp/scream_rrtmgp_interface.hpp"
+#include "physics/rrtmgp/tests/rrtmgp_test_utils.hpp"
+#include "cpp/rrtmgp/mo_gas_concentrations.h"
+#include "physics/rrtmgp/mo_garand_atmos_io.h"
+#include "cpp/rte/mo_fluxes.h"
+#include "cpp/extensions/cloud_optics/mo_cloud_optics.h"
+#include "physics/rrtmgp/simple_netcdf.hpp"
+#include <netcdf.h>
+
+namespace rrtmgpTest {
+
+    bool file_exists(const char *filename) {
+        if (auto file = fopen(filename, "r")) {
+            fclose(file);
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    // TODO: use YAKL intrinsics for this; this won't work on the GPU
+    bool all_equals(real2d &arr1, real2d &arr2) {
+        double tolerance = 0.01;
+        /*
```

Comment:
Do we want to just delete these commented lines?

Reply:
I left them in hopes of revisiting this to provide a GPU-friendly assertion. Right now I need to make host copies to do this check.

Reply:
Removed.